.github/workflows/build-addon-on-push.yml

```diff
@@ -3,6 +3,11 @@ name: Build on push
 
 on: [push, pull_request]
 
+env:
+  # Only used for the cache key. Increment version to force clean build.
+  GODOT_BASE_BRANCH: master
```

Comment:
Which version is the comment referring to?

Reply:
It's the version of the plugin really,  but since I'm reusing the action from godot-cpp (which copied it from Godot), it has that name.

Anyway, if we use this on another branch (other than `master`) we'll want to change that to the name of the branch